charts/varnish/templates/configmap-nginx.yaml

```diff
@@ -4,5 +4,38 @@ metadata:
   name: {{ include "varnish.fullname" . }}-nginx
   labels:
     {{- include "varnish.labels" . | nindent 4 }}
+{{- if .Values.nginx.directorCustomConfig -}}
+data:
+  nginx.conf: |-
+    events {}
+    http {
+      server { # simple reverse-proxy
+        listen       81;
+        location /health {
+          return 200 'status: ok';
+          access_log off;
+        }
+        # pass requests for dynamic content to rails/turbogears/zope, et al
+        location / {
```

Comment:
Is the config working on a "first-match" approach? If so - then `location /` will match all requests and the other location matches will never trigger. I'm not sure what nginx does there...

Reply:
good catch I will fix this